concepts/0003-protocols/semver.md

```diff
@@ -1,93 +1,51 @@
-# Semver Rules for Protocols
+### Semver Rules for Protocols
 
-[Semver](https://semver.org) rules apply in cascading fashion to versions
-of protocols and individual message types. The version of a message type
-or protocol is expressed in the `semver` portion of its [identifying URI](
-uris.md).
+[Semver](https://semver.org) rules apply to protocols, with the version of a protocol is expressed
+in the `semver` portion of its [identifying URI](#message-type-and-protocol-identifier-uris).
+The ["ingredients"](#ingredients) of a protocol combine to form a
+[public API in the semver sense](https://semver.org/#spec-item-1). Core Aries protocols
+specify only major and minor elements in a version; the patch component is not used.
 
-Individual message types are versioned as part of a coherent protocol, which
-constitutes a [public API in the semver sense](https://semver.org/#spec-item-1).
-An individual message type can add new optional fields, or deprecate
-existing fields, [with only a change to its protocol's minor
-version](https://semver.org/#spec-item-7).
-Similarly, a protocol can add new message types (or [adopted
-ones](../../0000-template-protocol.md#adopted-messages)) with only a change
-to the minor version. It can announce deprecated fields. It can add additional
-semantics around optional decorators. These are all backwards-compatible
-changes, also requiring only a minor version update.
+The major and minor versions of protocols match semvar semantics:
 
-Changes that remove fields or message types, that make formerly optional
+- Clarification updates that do not change the "public API" of the protocol can be made
+with community support _without_ changing the minor version of the protocol.
+- An individual message type can add new optional fields, or deprecate
+existing fields, [with only a change to its protocol's minor
+version](https://semver.org/#spec-item-7). Similarly, a protocol can
+add new or deprecate message types with only a change to the minor version.
+- Changes that remove fields or message types, that make formerly optional
 things required, or that alter the state machine in incompatible
 ways, must result in an [increase of the major version of the protocol/primary
 message family](https://semver.org/#spec-item-8).
 
-Because protocol handling choices depend mainly on major and minor version
```

Comment:
Although I agree that core protocols use only major.minor, I am not sure we should delete this paragraph. Are we saying that it's impossible to use patch with protocols -- or only that we don't use it in our our work? Do we want agents to support patch properly, or not? If not, we should explicitly say so. The previous statement about it not being used in core protocols is not sufficient to make this a global characteristic.

Reply:
Good catch on the distinction. My preference would be to eliminate them altogether vs. just for the core protocols,

In thinking about it, we really don't care either way.  If an agent is going to support a non-core protocol, it is going to get the protocol requirements for from some other place, and those requirements will indicate if support for handling a patch in the type is needed or not.  Adding that support might require changing common code for message type processing, but so be it.

I'm open to putting whatever is appropriate there -- e.g. "Core protocols must not use the patch component of a semver designation, but other protocols may include patch and handling should be included".

Reply:
Clarification added.